Model/Sales/Repository/PackageRepository.php

```diff
@@ -206,7 +206,7 @@ private function getValueFromAttribute($connection, string $tableName, string $a
      *
      * @return PackageRepository
      */
-    public function isDeliveryOptionsDisabled(array $products): self
+    public function isDeliveryOptionsDisabled($products): self
```

Comment:
Waarom is dit weg?

Reply:
products is een object en geen array. De object type is pas beschikbaar vanaf php 7.2 maar wij moeten ook nog php 7.0 ondersteunen.